clarity/Cargo.toml

```diff
@@ -23,7 +23,7 @@ rand_chacha = "=0.2.2"
 serde = "1"
 serde_derive = "1"
 serde_stacker = "0.1"
-regex = "1"
+regex = "~1.8"
```

Comment:
Please avoid using tilde, and use a precise version.  We want all nodes everywhere to have the exact same code.

Reply:
I though we are using [Cargo.lock](https://github.com/stacks-network/stacks-blockchain/blob/master/Cargo.lock) for this.